src/store/allIncidentsSlice/easyMode.js

```diff
@@ -46,7 +46,7 @@ export const useEasyMode = () => {
 
   const easyMode = {
     fetchIncidents: () => {
-      dispatch(fetchIncidents.actionCreator());
+      dispatch(fetchIncidents.actionCreator({type: 'approved'}));
```

Comment:
Notice that this line doesn't include spacing between your braces and the key: value found within them`{ item...`. Please ensure that we're using the formatter installed within our application.